### DISCUSSION TITLE
Power Meter Daten MQTT

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#1114] [Q&A] [answered]
Ich sehe den Wald vor lauter Bäumen nicht  🤦‍♂️ 

FRAGE: Liefert die OpenDTU-OnBattery die Power-Meter Werte per MQTT weiter an z.B. Home Assistant?

Laut Wiki WEB-API die OpenDTU-OnBattery liefert Power-Meter Daten 👍 
siehe https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Web-API
 "power_meter": {
        "enabled": true,
        "Power": {
            "v": 0,
            "u": "W",
            "d": 1
        }

Laut Wiki MQTT aber, finde ich keine Information 🤔  
siehe https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/MQTT
habe ich etwas verpasst?

**Comment by spcqike** [comment]
wie es mit HA aussieht, keine Ahnung, aber ja, per MQTT kommen die Werte raus.

ich hab nur nen Shelly3EM und frage nur die Gesamtleistung ab, das sieht dann so aus
![grafik](https://github.com/user-attachments/assets/91eca45a-9353-4f17-8728-31a09b3a20d3)

**Comment by schlimmchen** [comment, marked as answer]
HA autodiscovery für den PowerMeter fehlt leider gänzlich.
